src/main/java/com/cinema/service/UserService.java

```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
```

Comment:
but for this method it is ok to return Optional from service

Reply:
left as is